platform/testing/liquid-templates/README.md

```diff
@@ -5,13 +5,158 @@ The Liquid Template Unit Testing Framework was created to replace Cypress for un
+
+##### `color-contrast`
+
+The `CSS` file for the static pages found on VA.gov is generated during the build process and, therefore, cannot be referenced by the `HTML` files generated by this tool. The `color-contrast` check has been disabled because it requires the `CSS` file.
+
+##### `document-title`
+
+The `document-title` check has been disabled because `liquid` template `include` files (not entire page templates) can also be tested by this tool and those templates are missing the `title` tag (along with the `html`, `head`, and `body` sections, etc., etc., etc.).
+
+Here's the problem:
+
+If you want to test the `breadcrumbs` `include` file (`src/site/includes/breadcrumbs.drupal.liquid`) the `HTML` used to instantiate a new `JSDOM` object looks like this:
+
+```html
+<nav data-template="includes/breadcrumbs" aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
+  <!-- lots of liquid code here -->
+</nav>
+```
+
+The resulting `html` document looks like this:
+
+```html
+<html>
+<head></head>
+<body>
+  <nav data-template="includes/breadcrumbs" aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
+    <!-- lots of liquid code here -->
+  </nav>
+</body>
+</html>
+```
+
+Just like a browser, `JSDOM` adds whatever `HTML` is missing to make the document valid.
+
+Notice there's no `title` tag in the `head` of the document. Because of this, we can't perform Axe Checks to make sure a `title` is present, so we need to disable the `document-title` check.
+
+In addition to the reason outlined above, we experienced false-positives on pages that had a valid `title` tag when running the `document-title` check. 
+
+If you want to test the presence of a `title` tag, and that the `title` isn't an empty string, you'll have write unit tests for those 'the old fashion way'.
+
+#### How to Run an Axe Check
+
+Run an accessibility check like this:
+
+```js
+it('reports no axe violations', async () => {
+  const violations = await axeCheck(container);
+  expect(violations.length).to.equal(0);
+});
+```
+
+Accessibility violations are logged to the console and look like this:
+
+```javascript
+6 Accessibility Violations Were Detected
+
+Axe Violation 1:
+ {
+  id: 'aria-roles',
+  impact: 'critical',
+  tags: [ 'cat.aria', 'wcag2a', 'wcag412' ],
+  description: 'Ensures all elements with a role attribute use a valid value',
+  help: 'ARIA roles used must conform to valid values',
+  helpUrl: 'https://dequeuniversity.com/rules/axe/4.1/aria-roles?application=axeAPI',
+  nodes: [
+    {
+      any: [],
+      all: [],
+      none: [Array],
+      impact: 'critical',
+      html: '<h1 role="nonsense">VA Pittsburgh health care</h1>',
+      target: [Array],
+      failureSummary: 'Fix all of the following:\n' +
+        '  Role must be one of the valid ARIA roles: nonsense'
+    }
+  ]
+}
```

Comment:
Maybe we can wrap the code snippets in a `<details>` element so we can toggle it?

Reply:
Great idea. I just updated that.